src/main/java/com/paneedah/mwc/items/guns/M9A1Factory.java

```diff
@@ -21,588 +17,489 @@
 import net.minecraft.util.math.Vec3d;
 import org.lwjgl.opengl.GL11;
 
-import java.util.Arrays;
-
 public class M9A1Factory implements GunFactory {
 
     public Item createGun(CommonProxy commonProxy) {
         return new Weapon.Builder()
+                .withName("m9a1")
+                .withFireRate(0.5f)
+                .withRecoil(5f)
```

Comment:
No balance changes until 0.2.
```suggestion
                .withRecoil(2f)
```